.env.example

```diff
@@ -97,6 +97,7 @@ PERIODICAL=stop_expired_deploys:60,remove_expired_locks:60,report_system_stats:6
 # SLACK_CLIENT_ID= # optional, see plugins/slack_app/README.md
 # SLACK_CLIENT_SECRET= # optional, see plugins/slack_app/README.md
 # SLACK_VERIFICATION_TOKEN= # optional, see plugins/slack_app/README.md
+# SLACK_GLOBAL_BUDDY_REQUEST="<webhook_url>#<channel>" # send notifications for all buddy requests to this channel
```

Comment:
yeah I know ... make a better suggestion :D 
`:` is already in `http://` ... `;` is weird ... `@` might collide with basic auth ... `,` is for multiple entries